apps/meteor/client/lib/chats/flows/sendMessage.ts

```diff
@@ -55,14 +61,17 @@ export const sendMessage = async (
 
 	chat.readStateManager.clearUnreadMark();
 
+	const uploadsStore = tmid ? chat.threadUploads : chat.uploads;
```

Comment:
Instead of accessing the uploadsStore here, load everything you need from it and pass it as arguments to `sendMessage`, then you can also clear the store before calling sendMessage.